databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -394,16 +394,22 @@ def delete_share_cli(api_client, name):
               help=(
                   'IP address in CIDR notation that is allowed to use delta sharing. '
                   '(can be specified multiple times).'))
+@click.option('--property', default=None, type=JsonClickType(), required=False, multiple=True,
```

Comment:
Maybe `--property key value` or `--property key=value` would be easier for users than specifying a JSON each time.

https://stackoverflow.com/questions/51164033/python-click-multiple-key-value-pair-arguments

Though the benefit of the JSON is supporting multiple future properties, but are we planning on other properties to the JSON?

Reply:
Good idea. I changed it to be separated by the equal sign `key=value` with a caveat that it won't support setting `=` as part of the key or value which I think is probably fine.